resources/views/pages/home.blade.php

```diff
@@ -112,7 +112,8 @@
 
 
 
-{{-- <section class="py-8 bg-white md:py-20 lg:py-40">
+@if(count($posts) > 0)
```

Comment:
@lpheller like so?

Reply:
Yep, exactly. Probably could do it like this as well:
```suggestion
@if(!$posts->isEmpty())
// or
@if($posts->isNotEmpty()
```
but that shouldn't matter, imo it just reads a little nicer.

Reply:
Or
```suggestion
@if($posts->isNotEmpty())
```

Reply:
I added the ->isNotEMpty() method